config/locales/devise.en.yml

```diff
@@ -45,8 +45,8 @@ en:
       update_needs_confirmation: "You updated your account successfully, but we need to verify your new email address. Please check your email and follow the confirm link to confirm your new email address."
       updated: "Your account has been updated successfully."
     sessions:
-      signed_in: "Signed in successfully."
-      signed_out: "Signed out successfully."
+      signed_in: ''
```

Comment:
If we no longer need these messages in the app (`signed_in` and `signed_out`) I think it's best to delete them from this file entirely.